Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -472,23 +472,14 @@ public async Task RequestHandler(IHttpRequest httpReq, string urlString, string
                 urlToLog = GetUrlToLog(urlString);
 
                 if (string.Equals(localPath, "/" + _config.Configuration.BaseUrl + "/", StringComparison.OrdinalIgnoreCase)
-                    || string.Equals(localPath, "/" + _config.Configuration.BaseUrl, StringComparison.OrdinalIgnoreCase))
+                    || string.Equals(localPath, "/" + _config.Configuration.BaseUrl, StringComparison.OrdinalIgnoreCase)
+                    || string.Equals(localPath, "/", StringComparison.OrdinalIgnoreCase)
+                    || string.IsNullOrEmpty(localPath))
                 {
                     httpRes.Redirect("/" + _config.Configuration.BaseUrl + "/" + _defaultRedirectPath);
                     return;
                 }
 
-                if (string.Equals(localPath, "/", StringComparison.OrdinalIgnoreCase))
-                {
-                    httpRes.Redirect(_defaultRedirectPath);
```

Comment:
Just to preemptively comment here, this made little sense to be being separate. I assume it was to handle relative links? In any case, I made all these redirects full links with this change.